src/WPConfigTransformer.php

```diff
@@ -89,7 +89,7 @@ public function exists( $type, $name ) {
 	 * @param string $type Config type (constant or variable).
 	 * @param string $name Config name.
 	 *
-	 * @return array
+	 * @return string|null
```

Comment:
```suggestion
	 * @return mixed
```

Maybe this would be a little safer?

Reply:
I was thinking the same originally, but hinting nullable string makes it clear that the method does not cast the value to it's type, and maybe to some extent that undefined variables make it return null 🙂

Reply:
Works for me 👍